test/Formik.test.tsx

```diff
@@ -470,13 +481,24 @@ describe('<Formik>', () => {
         await wait(() => expect(onSubmit).toBeCalled());
       });
 
-      it('should not submit the form if invalid', () => {
+      it('should not submit the form if invalid', async () => {
         const onSubmit = jest.fn();
-        const validate = jest.fn(() => Promise.resolve({ name: 'Error!' }));
+        const validate = jest.fn(() => Promise.reject({ name: 'Error!' }));
```

Comment:
This was also "wrong", we want to check that the form is not submitted when `validate` rejects the promise